suites/test_module_edit_publish.py

```diff
@@ -17,14 +17,17 @@
 @lcc.suite(description="Suite: Verify that authenticated user can edit metadata and publish module", rank=1)
 class test_module_edit_publish:
   api_auth = lcc.inject_fixture("api_auth")
+  path_for_module = ""
+  request_url = ""
 
   @lcc.test("Verify that authenticated user can edit metadata successfully")
   def edit_metadata(self, api_auth, setup_test_products):
     variant = utilities.read_variant_name_from_pantheon2config()
     lcc.log_info(str(variant))
     variant = str(variant)
-    path_for_module = utilities.select_first_item_from_search_results(fixture.url, module_title_prefix)
-    edit_metadata_url = fixture.url + "content/" + path_for_module + "/en_US/variants/" +\
+    test_module_edit_publish.path_for_module = utilities.select_first_module_from_search_results(fixture.url,
```

Comment:
hey @noopurmath3 I think self.path_for_module would simply work too. :) you can refer the assembly tests too.